practices/bad_team_onboarding.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: agile-events
+stages:
+  - development
+short_description: The bad team onboarding is when your new members don't receive the crucial information or vice-versa i.e. they are overflooded with the pointers and information at the pace.
+tags:
+  - retrospective
+  - refinement
+  - demotivated team
+  - less productivity
+keywords:
+  - demotivated team
+  - retrospective
+  - meaningless work
+---
+
+# Bad Team Onboarding
+
+**TL;DR**
+
+A great onboarding process should help an employee identify with the company and feel a sense of belonging–like joining a family that welcomes you with open arms, but when this feeling is lost somewhere it leds to Bad Team Onboarding
```

Comment:
Hello, this sentence is too similar to this one: _A great onboarding process should help an employee identify with the company and feel a sense of belonging–like joining a family that welcomes you with open arms._  from the https://www.jobvite.com/wp-content/uploads/2016/08/Jobvite_eBook_Onboarding_Gone_Wrong.pdf. 

Please, rewrite it with your own words.